powerauth-push-server/src/main/java/io/getlime/push/controller/rest/AdministrationController.java

```diff
@@ -165,13 +171,36 @@ public Response updateIos(@RequestBody ObjectRequest<UpdateIosRequest> request)
         return new Response();
     }
 
+    /**
+     * Update APNs configuration.
+     * @param request Update APNs configuration request.
+     * @return Response.
+     * @throws PushServerException Thrown when application credentials entity could not be found or request validation fails.
+     */
+    @RequestMapping(value = "apns/update", method = { RequestMethod.POST, RequestMethod.PUT })
```

Comment:
I am not sure I understand what you mean by this comment.

Reply:
Apologies, it seems I accidentally replaced the text with the pasting of the example.

We can leave the REST API pattern as-is. 
But we can change to a more standard approach - at least for the new (non-deprecated API)
i.e., `POST /admin/app/apns` instead of `POST /admin/app/apns/update`
With the following changes to the remove API, i.e., `DELETE /admin/app/apns` instead of `DELETE /admin/app/apns/remove`

Reply:
OK, I can change it.

Reply:
For the `DELETE` methods, we will need to use `@RequestParam String appId` because request bodies may not be always supported for `DELETE` methods and the `POST` method is removed.